package-source.sh

```
@@ -0,0 +1,70 @@
+# Scripts
+cp package-extension.sh source-package/
+cp package-source.sh source-package/
+
+# Documentation and legal files
+cp README.md source-package/
+cp PRIVACY_POLICY.md source-package/
+cp privacy-policy.html source-package/
+
+# Create the source code ZIP file
+echo "Creating source code ZIP package..."
+cd source-package
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)